### ISSUE TITLE
`minikube tunnel` fails with "loadbalancer_patcher.go:108] error patching <service> with IP"

### ISSUE THREAD

**Issue by paultiplady** [kubernetes/minikube#3940]
Running `minikube tunnel` on MacOS with a single service gives:

```
$ sudo minikube tunnel
E0322 22:22:04.059782   51591 loadbalancer_patcher.go:108] error patching monolith with IP 10.110.55.40: %!s(<nil>)
Status:
        machine: minikube
        pid: 51591
        route: 10.96.0.0/12 -> 192.168.64.4
        minikube: Running
        services: [monolith]
    errors: 
                minikube: no errors
                router: no errors
                loadbalancer emulator: no errors
```

OS version 10.14.3 (18D109)
Minikube version v0.35.0

**Comment by paultiplady** [comment]
It does look like a new route has been added: 

```
$ netstat -nr -f inet
...
10.96/12           192.168.64.4       UGSc            0        0 bridge1                                               
...
```

And my service has the IP:

```
$ kubectl get svc monolith                
NAME       TYPE           CLUSTER-IP     EXTERNAL-IP    PORT(S)                      AGE
monolith   LoadBalancer   10.110.55.40   10.110.55.40   443:30323/TCP,80:31472/TCP   42m
```

But connecting to `https://10.110.55.40` doesn't do anything.

**Comment by balopat** [comment]
Yes, this is a spurious error - it was fixed in https://github.com/kubernetes/minikube/commit/229137c06b5ee0d210676bb23114912a689e1893 
and released with 1.0.0. 
Please upgrade and check! 
Closing.
